Source/ASVideoNode.mm

```diff
@@ -101,6 +101,9 @@ - (instancetype)initWithCache:(id<ASImageCacheProtocol>)cache downloader:(id<ASI
   [self addTarget:self action:@selector(tapped) forControlEvents:ASControlNodeEventTouchUpInside];
   _lastPlaybackTime = kCMTimeZero;
   
+  NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
+  [notificationCenter addObserver:self selector:@selector(applicationDidBecomeActive:) name:UIApplicationDidBecomeActiveNotification object:nil];
```

Comment:
Let's be sure to remove ourself as an observer in `dealloc`. Heads up, it's best to remove using the specific method `removeObserver:name:object:` rather than simply `removeObserver:` which is much slower 😱